src/assets/scripts/client/aircraft/FlightManagementSystem/Fms.js

```diff
@@ -480,6 +481,70 @@ export default class Fms {
         return this._routeModel.absorbRouteModel(nextRouteModel);
     }
 
+    /**
+    * Return the next waypoint having an #altitudeMaximum equal to or less than the specified value
```

Comment:
the spacing on all the docblocks in this file are off by one missing space
```
// do not
/**
*
*/

// do
/**
 *
 */
```

Reply:
gah! done.